pages/_app.js

```diff
@@ -10,7 +10,7 @@ export default function Flossbank ({ Component, pageProps }) {
   return (
     <ThemeProvider theme={CustomTheme}>
       <CSSReset />
-      <ColorModeProvider>
+      <ColorModeProvider value='light'>
```

Comment:
chakra has bug where some components (modal) in prod will be dark theme even though rest of the site isn't. this fixes that